relationships/synthesis/INFRA-KUBERNETES_CONTAINER-to-EXT-SERVICE.yml

```diff
@@ -0,0 +1,33 @@
+relationships:
+  - name: k8sContainerEXTService
+    version: "1"
+    origins:
+      - OpenTelemetry
+    conditions:
+      - attribute: eventType
+        anyOf: [ "Metric" ]
+      - attribute: entity.type
+        anyOf: [ "CONTAINER" ]
+    relationship:
+      expires: P75M
+      relationshipType: HOSTS
+      source:
+        buildGuid:
+          account:
+            lookup: true
+          domain:
+            value: INFRA
+          type:
+            value: CONTAINER
+          identifier:
+            fragments:
+              - value: "k8s:"
```

Comment:
This looks like constructing a GUID for a pod, but the relationship name says `k8sContainerEXTService`.
Is this correct ?

Reply:
I'm trying to create a relationship between the infra container(Infra metrics reported by oTel) with the EXT service. 
Could you suggest me how to build GUID for the infra-container.

Reply:
I think you would need to reach out to Infra as they are owning the logic on how they generate that id.
From what I can see from these synthesis rules, it's probably this: https://github.com/newrelic/entity-definitions/blob/16437442e365acd609178341178acb06d5c5994b/entity-types/infra-container/definition.yml#L131